src/c++/library/grpc_client.h

```diff
@@ -57,6 +58,29 @@ struct SslOptions {
   std::string certificate_chain;
 };
 
+// GRPC KeepAlive: https://grpc.github.io/grpc/cpp/md_doc_keepalive.html
+struct KeepAliveOptions {
+  explicit KeepAliveOptions()
+      : keepalive_time_ms(INT_MAX), keepalive_timeout_ms(20000),
+        keepalive_permit_without_calls(false), http2_max_pings_without_data(2)
+  {
+  }
+  // The period (in milliseconds) after which a keepalive ping is sent on the
```

Comment:
on ->over

Reply:
These descriptions are taken directly from https://github.com/grpc/grpc/blob/master/doc/keepalive.md#keepalive-user-guide-for-grpc-core-and-dependents